config/develop/bgrande-nextflow-service-user.yaml

```diff
@@ -0,0 +1,12 @@
+# Provision a service account for AWS API access keys
+template_path: "remote/service-user.yaml"
+stack_name: "bgrande-nextflow-service-user"
+stack_tags:
+  Department: "CompOnc"
+  Project: "imCORE"
+  OwnerEmail: "bruno.grande@sagebase.org"
+parameters:
+  ManagedPolicyArns: ["arn:aws:iam::aws:policy/AmazonS3FullAccess"]
```

Comment:
you can be more specific about which bucket(s) this service user has access to by providing a custom managed or inline policy specifying the specific buckets.  here's an example.. https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/sceptre/sageit/config/prod/gha-svc-account.yaml

Reply:
Thanks for the example!